[Pages] Add CAA records to Custom Domains guide (#5312)
cloudflare/cloudflare-docs: content/pages/platform/custom-domains.md

3.  Select the **three dot icon** next to your custom domain > **Remove domain**.

After completing these steps, your Pages project will only be accessible through the `*.pages.dev` subdomain you chose when creating your project.

> ## Known Issues
> 
> ### CAA Records

Reviewer: ```suggestion
## Known issues

### CAA records
```

> CAA records allow you to restrict certificate issuance to specific Certificate Authorities (CAs).

Reviewer: ```suggestion
Certification Authority Authorization (CAA) records allow you to restrict certificate issuance to specific Certificate Authorities (CAs).
```

> This can cause issues when adding a custom domain to your Pages project if you have CAA records that don't allow Cloudflare to a certificate for your custom domain.

Reviewer 2: Should we explicitly mention external DNS here?

Author: Probably.

Cloudflare will add them automatically if you are using Universal SSL, AMP Real URL or Signed Exchanges & you add your own CAA records.

I guess there's super edge cases where you would have all of those disabled but at that point I'd be confused.

> This can cause issues when adding a custom domain to your Pages project if you have CAA records that don't allow Cloudflare to a certificate for your custom domain.

Reviewer: ```suggestion
This can cause issues when adding a [custom domain](/pages/platform/custom-domains/) to your Pages project if you have CAA records that do not allow Cloudflare to a certificate for your custom domain.
```

> This can cause issues when adding a [custom domain](/pages/platform/custom-domains/) to your Pages project if you have CAA records that do not allow Cloudflare to a certificate for your custom domain.

Reviewer: to ___ 

seems like it is missing something?

Author: Supposed to say issue

:notlikecatgoogly:

Author: ```suggestion
This can cause issues when adding a [custom domain](/pages/platform/custom-domains/) to your Pages project if you have CAA records that do not allow Cloudflare to issue a certificate for your custom domain.
```
